site/src/scss/_base.scss

```diff
@@ -21,9 +22,21 @@ pre code {
   font-weight: bold;
   padding: rem-calc(20) rem-calc(64);
 
+  @include breakpoint(large) {
+    min-width: rem-calc(291);
```

Comment:
I think this will no longer be an issue if we can not absolutely position the buttons below, but we shouldn't be constraining the width of these buttons in any way. We can make them expand to full on mobile if that's what this is for.

Reply:
This is to make the buttons wide enough on large to match with the designs. Once it breaks to medium, as you can see, it returns to the default max width.

Reply:
There is no absolute positioning on any buttons that I put on...

Reply:
Ok.  What I'm saying is that these buttons are spanning 1/3 of their container, so forcing them to be bigger than that is going to cause them to overflow the container.  So we should be dealing with the container sizing, not the button sizing.

Reply:
Yeah. I can see that now. They start to overlap each other and look busted before the breakpoint. I get it.